src/integrationtests/java/com/aws/greengrass/integrationtests/ConfigTest.java

```diff
@@ -386,4 +388,39 @@ void GIVEN_Greengrass_with_mqtt_bridge_WHEN_invalid_mqttTopicMapping_updated_THE
 
         assertTrue(bridgeErrored.await(AWAIT_TIMEOUT_SECONDS, TimeUnit.SECONDS));
     }
+
+    @SuppressWarnings("PMD.AvoidCatchingGenericException")
+    @TestWithMqtt5Broker
+    @WithKernel("config.yaml")
+    void GIVEN_bridge_WHEN_mqtt_version_toggled_THEN_clients_switched(Broker broker, ExtensionContext context) throws Exception {
```

Comment:
This test reproduced issues we see in UAT?

Reply:
yeah i observed "already connected" in logs and verified those go away with this test

Reply:
and as discussed still unsure about the shutdown timeout